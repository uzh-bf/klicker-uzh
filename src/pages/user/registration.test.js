/* eslint-env jest */

import * as React from 'react'
import renderer from 'react-test-renderer'

import Registration from './registration'

// HACK: workaround for https://github.com/Semantic-Org/Semantic-UI-React/issues/1702
jest.mock('react-dom', () => ({
  findDOMNode: jest.fn(() => {}),
}))

describe('Snapshot-Testing', () => {
  it.skip('Works', () => {
    const component = renderer.create(<Registration />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
