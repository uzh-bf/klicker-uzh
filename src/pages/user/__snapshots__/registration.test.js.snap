// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot-Testing Works 1`] = `
<div
  className="staticLayout"
  data-jsx={2161656035}
>
  <div
    className="content"
    data-jsx={2161656035}
  >
    <div
      className="registration"
      data-jsx={232182413}
    >
      <h1
        data-jsx={232182413}
      >
        <span>
          Registration
        </span>
      </h1>
      <form
        className="ui form error"
        data-jsx={1683376726}
        onSubmit={[Function]}
      >
        <div
          className="personal"
          data-jsx={1683376726}
        >
          <div
            className="field"
          >
            <label
              data-jsx={1632341904}
              forHtml="firstName"
            >
              First name
            </label>
            <input
              data-jsx={1632341904}
              name="firstName"
              onBlur={[Function]}
              onChange={[Function]}
              onDragStart={[Function]}
              onDrop={[Function]}
              onFocus={[Function]}
              placeholder="First name"
              type="text"
              value=""
            />
          </div>
          <div
            className="field"
          >
            <label
              data-jsx={1632341904}
              forHtml="lastName"
            >
              Last name
            </label>
            <input
              data-jsx={1632341904}
              name="lastName"
              onBlur={[Function]}
              onChange={[Function]}
              onDragStart={[Function]}
              onDrop={[Function]}
              onFocus={[Function]}
              placeholder="Last name"
              type="text"
              value=""
            />
          </div>
          <div
            className="required field"
          >
            <label
              data-jsx={1632341904}
              forHtml="email"
            >
              Email
            </label>
            <input
              data-jsx={1632341904}
              name="email"
              onBlur={[Function]}
              onChange={[Function]}
              onDragStart={[Function]}
              onDrop={[Function]}
              onFocus={[Function]}
              placeholder="Email"
              type="email"
              value=""
            />
          </div>
        </div>
        <div
          className="account"
          data-jsx={1683376726}
        >
          <div
            className="required field"
          >
            <label
              data-jsx={1632341904}
              forHtml="shortname"
            >
              Shortname
            </label>
            <input
              data-jsx={1632341904}
              name="shortname"
              onBlur={[Function]}
              onChange={[Function]}
              onDragStart={[Function]}
              onDrop={[Function]}
              onFocus={[Function]}
              placeholder="Shortname"
              type="text"
              value=""
            />
          </div>
          <div
            className="required field"
          >
            <label
              data-jsx={1632341904}
              forHtml="password"
            >
              Password
            </label>
            <input
              data-jsx={1632341904}
              name="password"
              onBlur={[Function]}
              onChange={[Function]}
              onDragStart={[Function]}
              onDrop={[Function]}
              onFocus={[Function]}
              placeholder="Password"
              type="password"
              value=""
            />
          </div>
          <div
            className="required field"
          >
            <label
              data-jsx={1632341904}
              forHtml="passwordRepeat"
            >
              Repeat password
            </label>
            <input
              data-jsx={1632341904}
              name="passwordRepeat"
              onBlur={[Function]}
              onChange={[Function]}
              onDragStart={[Function]}
              onDrop={[Function]}
              onFocus={[Function]}
              placeholder="Repeat password"
              type="password"
              value=""
            />
          </div>
          <div
            className="field"
            data-jsx={1683376726}
          >
            <label
              data-jsx={1683376726}
              htmlFor="useCase"
            >
              <span>
                Use case description
              </span>
            </label>
            <textarea
              name="useCase"
              onBlur={[Function]}
              onChange={[Function]}
              onDragStart={[Function]}
              onDrop={[Function]}
              onFocus={[Function]}
              type="text"
              value=""
            />
          </div>
          <button
            className="ui primary disabled right floated button"
            disabled={true}
            onClick={[Function]}
            role="button"
            tabIndex={-1}
            type="submit"
          >
            <span>
              Submit
            </span>
          </button>
        </div>
      </form>
    </div>
  </div>
  <footer
    data-jsx={2161656035}
  >
    Â© IBF
  </footer>
</div>
`;
